test/runtests.jl

```diff
@@ -317,8 +345,8 @@ cd(@__DIR__) do
         Test.pop_testset()
     end
     println()
-    Test.print_test_results(o_ts,1)
-    if !o_ts.anynonpass
+    Test.print_test_results(o_ts, 1)
+    if !o_ts.anynonpass && skipped == 0
```

Comment:
I think this `skipped == 0` should be covered by the `completed_tests` case which inserts a `Test.Error` for each skipped test. So probably not required unless I'm missing something.

Reply:
Ah, thanks for explaining that, it makes sense now.